static/js/settings_bots.js

```diff
@@ -185,9 +199,24 @@ exports.set_up = function () {
         var bot_type = $('#create_bot_type :selected').val();
         // If the selected bot_type is Outgoing webhook
         if (bot_type === OUTGOING_WEBHOOK_BOT_TYPE) {
+            // For "outgoing webhook" bot type.
             $('#payload_url_inputbox').show();
             $('#create_payload_url').addClass('required');
+
+            // For "embedded bot" bot type.
+            $('#service_name_list').hide();
+            $('#select_service_name').removeClass('required');
+
+        } else if (bot_type === EMBEDDED_BOT_TYPE) {
+            $('#service_name_list').show();
+            $('#select_service_name').addClass('required');
+
+            $('#payload_url_inputbox').hide();
+            $('#create_payload_url').removeClass('required');
         } else {
```

Comment:
Actually it is `else if (bot_type === GENERIC_BOT_TYPE || bot_type === INCOMING_WEBHOOK)`. Since it handles the remaining 2 cases, I have simply added a comment in the code.
Makes sense?